protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -122,6 +122,25 @@ definitions:
           - SOURCE_RETRIEVAL_ERROR
           # Errors casting to appropriate type
           - DESTINATION_TYPECAST_ERROR
+  AirbyteFileTransferMessage:
+    type: object
+    additionalProperties: true
+    required:
+      - stream
+      - file
+      - emitted_at
+    properties:
+      namespace:
```

Comment:
nit: It feels weird to me to see fields like `namespace` and `stream` (which I assume is `stream_name`) as it feels like the [StreamDescriptor](https://github.com/airbytehq/airbyte-protocol/blob/009278cfa8677e1bfdfaeb7cabfc8415c06aef34/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml#L222-L231) concept we already have. Why don't we re-use it?

Reply:
> It feels weird to me to see fields like namespace
Revisiting `namespace` is not needed; I will remove it; thanks for making me give a second check on this.

> stream (which I assume is stream_name) as it feels like the [StreamDescriptor](https://github.com/airbytehq/airbyte-protocol/blob/009278cfa8677e1bfdfaeb7cabfc8415c06aef34/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml#L222-L231) concept we already have. Why don't we re-use it?

Well, we could, but I think it would require us to make changes on the platform (and destination?) side, as the current structure is similar to a record, where stream is a string rather than an object.

File record sample:

```
{
  "type": "RECORD",
  "record": {
    "stream": "random_data_10",
    "file": {
      "file_url": "/tmp/airbyte-file-transfer/5GB_Files/random_data_10_1.csv",
      "bytes": 996913046,
      "file_relative_path": "5GB_Files//random_data_10_1.csv",
      "modified": 1729720256000,
      "source_file_url": "5GB_Files//random_data_10_1.csv"
    },
    "emitted_at": 1731446522971,
    "data": {}
  }
}
```

Regular record sample
```
{
  "type": "RECORD",
  "record": {
    "stream": "random_data_10",
    "data": {
      "name": "Daniel Johnson",
      "age": 36,
      "state": "California",
      "nationality": "Albania",
      "race": "HoneyDew",
      "_ab_source_file_last_modified": "2024-10-23T15:50:56.000000Z",
      "_ab_source_file_url": "5GB_Files//random_data_10_1.csv"
    },
    "emitted_at": 1731512134588
  }
}
```

Stream descriptor from state message:

`{"stream_descriptor":{"name":"random_data_10"}`

Currently [StreamDescriptor](https://github.com/airbytehq/airbyte-protocol/blob/009278cfa8677e1bfdfaeb7cabfc8415c06aef34/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml#L222-L231) is used for `AirbyteStreamState`, `AirbyteErrorTraceMessage`, `AirbyteStreamStatusTraceMessage`. It would depend on platform expectations if we can simply change this or coordinate changes.


cc @benmoriceau @tryangul